### PR TITLE
Document IMAP methods to retrieve preview information

### PR DESCRIPTION
**Pull request by slusarz**

### CODE REVIEW

File: source/configuration_manual/imap_preview.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.. _imap_preview:
+
+============
+IMAP PREVIEW
```
Comment (sirainen) [inline source/configuration_manual/imap_preview.rst:4]: This is definitely not configuration_manual. Possibly admin_manual?

Comment (slusarz) [reply source/configuration_manual/imap_preview.rst:4]: Moved to admin_manual

**Comment by cmouse** [comment]
PREVIEW is available already in 2.3.13 and prior to that too.

**Comment by cmouse** [comment]
To my knowledge nothing has been deprecated so far.

**Review by cmouse — APPROVED** [review]

**Pull request merged**